mule-extensions-api/src/main/java/org/mule/runtime/extension/api/model/AbstractExecutableComponentModel.java

```diff
@@ -260,6 +262,58 @@ protected AbstractExecutableComponentModel(String name,
                                              DeprecationModel deprecationModel,
                                              SampleDataProviderModel sampleDataProviderModel,
                                              Set<String> semanticTerms) {
+    this(name, description, parameterGroupModels, output, outputAttributes, requiresConnection, transactional, supportsStreaming,
+         displayModel, errors, stereotype,
+         visibility, modelProperties, nestedComponents, notifications, deprecationModel, sampleDataProviderModel, semanticTerms,
+         null);
```

Comment:
same comment as in the mule-api PR. I would avoid the null/optional pattern and instead default to 4.1.1